models/attribute-definition/src/builder.spec.ts

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable jest/no-disabled-tests */
+/* eslint-disable jest/valid-title */
+import { createBuilderSpec } from '@commercetools-test-data/core/test-utils';
+import { TAttributeDefinition, TAttributeDefinitionGraphql } from './types';
+import * as AttributeDefinition from '.';
+
+describe('builder', () => {
+  it(
+    ...createBuilderSpec<TAttributeDefinition, TAttributeDefinition>(
+      'default',
+      AttributeDefinition.random(),
+      expect.objectContaining({
+        type: expect.objectContaining({ name: expect.any(String) }),
+        name: expect.any(String),
+        label: expect.objectContaining({
+          de: expect.any(String),
+          en: expect.any(String),
+          fr: expect.any(String),
+        }),
+        isRequired: expect.any(Boolean),
+        attributeConstraint: expect.any(String),
+        inputTip: expect.objectContaining({
+          de: expect.any(String),
+          en: expect.any(String),
+          fr: expect.any(String),
+        }),
+        inputHint: expect.any(String),
+        isSearchable: expect.any(Boolean),
+      })
+    )
+  );
+
+  it(
+    ...createBuilderSpec<TAttributeDefinition, TAttributeDefinition>(
+      'rest',
+      AttributeDefinition.random(),
+      expect.objectContaining({
+        type: expect.objectContaining({ name: expect.any(String) }),
+        name: expect.any(String),
+        label: expect.objectContaining({
+          de: expect.any(String),
+          en: expect.any(String),
+          fr: expect.any(String),
+        }),
+        isRequired: expect.any(Boolean),
+        attributeConstraint: expect.any(String),
+        inputTip: expect.objectContaining({
+          de: expect.any(String),
+          en: expect.any(String),
+          fr: expect.any(String),
+        }),
+        inputHint: expect.any(String),
+        isSearchable: expect.any(Boolean),
+      })
+    )
+  );
+
+  it(
+    ...createBuilderSpec<TAttributeDefinition, TAttributeDefinitionGraphql>(
+      'graphql',
+      AttributeDefinition.random(),
+      expect.objectContaining({
+        type: expect.objectContaining({
+          name: expect.any(String),
+          __typename: expect.any(String),
```

Comment:
We know what the `typename` is here and elsewhere, so might as well be specific. This comment applies to all the `typename` fields in tests.

Reply:
This `__typename` seems to change based on the `random` call, but the others that are built with `addFields` 
(like `__typename: "LocalizedString",`)
don't, do I have that right, or do I need to specify in the call to `random`?